3rdparty/python/requirements.txt

```diff
@@ -25,5 +25,5 @@ scandir==1.2
 setproctitle==1.1.10
 setuptools==30.0.0
 six>=1.9.0,<2
-thrift==0.9.1
+thrift>=0.9.1
```

Comment:
I think <0.10 should be fine. I had to change this because it failed to resolve 0.9.1 (even though it is available on PyPi - so I'm guessing there was a version clash with some dep).
 However I think the wire-format change happened between point releases, so all semver bets may be off.

I don't mind pinning this to, say, 0.9.3, or whatever it resolved to in practice, if you think that's preferable.